### ISSUE TITLE
Steam: Update minimum space required

### ISSUE THREAD

**Issue by MCOfficer** [endless-sky/endless-sky#4953]
**Describe the bug**
The README tells us ES needs at least 65MB free disk space, Steam says it's 75 MB. Seeing how the releases as well as the repository are already bordering on 100MB, this cannot be accurate.

**Comment by tehhowch** [comment]
Updated the readme to 120/300 https://github.com/endless-sky/endless-sky/commit/32646e04f607de323909f6a5bd775b737a75d33c

**Comment by MCOfficer** [comment]
Seems like either MZ or tehh updated this already; steam now shows 120MB across all OSes.

**Comment by quyykk** [comment]
The current README says that 200MB is needed, so the Steam requirements are still out of date. Also 200MB seems kinda low, I wouldn't be surprised if 0.9.15 will need a lot of space.

**Comment by MCOfficer** [comment]
You're right, continuous is already up to 190MB (uncompressed), so 120MB miminum won't cut it. Let's bump that to 200MB miminum, 300MB recommended?

**Comment by quyykk** [comment]
Sounds fine. I also checked RAM usage and on a clean install of continuous I get a bit than 1GB on Linux. Could you confirm? If so then we'll need to increase that requirement as well.

**Comment by MCOfficer** [comment]
1GB commited memory, ~200MB actual on windows. I wonder which number is more relevant for us, but I don't think it matters much - good luck trying to find a machine with OpenGL 3 but less than 1GB RAM.

**Comment by tehhowch** [comment]
For RAM we should definitely bump that--the huge animation sequences added in the past 2-ish years require some nice contiguous chunks to allocate (high-dpi even requires 64-bit game to not crash).

Probably the best question to ask is: how should we come up with these numbers? What criteria constitute minimum vs recommended? And what should we do if the minimum becomes outrageous?

Updating this metadata every release is basically required, so we should consolidate the procedure into e.g. a releasing document and consider that task completion as justification for closing this issue.

**Comment by quyykk** [comment]
> how should we come up with these numbers? What criteria constitute minimum vs recommended?

I would suggest using a clean (no plugins) ES for the minimum requirements, and then round the numbers up a bit. For the recommended I would do the same but with the high dpi plugin installed.
